pkg/serve/index.html

```diff
@@ -120,7 +120,31 @@ <h1 class="title">Skenario</h1>
             </div>
 
             <hr>
+            <div class="field is-horizontal">
+                <div class="field-label is-normal">
+                    <label class="label" for="requestTimeout">Request timeout (seconds)</label>
+                </div>
+                <div class="control">
+                    <input type="number" style="width: 5em" id="requestTimeout" value="10.0" min="1" step="1"/>
+                </div>
+            </div>
 
+            <div class="field is-horizontal">
+                <div class="field-label is-normal">
+                    <label class="label" for="requestCPUUtilization">Request CPU utilization </label>
```

Comment:
CPU utilization isn't quite what we discussed.  As a pod is given larger or smaller resource requests (VPA) then utilization will change for a give request.  I think what we need to pin here is the CPU time (proxy for cycles) the request needs to complete.

Reply:
Changed it. Please see 935d1a2c2d76ab2acf813d2b062574cf7d349078 commit.